src/libLookup/Lookup.cpp

```diff
@@ -1170,9 +1183,14 @@ bool Lookup::ProcessGetDSInfoFromSeed(const bytes& message, unsigned int offset,
     }
   }
 
-  uint128_t ipAddr = from.m_ipAddress;
-  Peer requestingNode(ipAddr, portNo);
-  P2PComm::GetInstance().SendMessage(requestingNode, dsInfoMessage);
+  Peer requestingNode(from.m_ipAddress, portNo);
+  unsigned char startByteResponse = START_BYTE_NORMAL;
```

Comment:
I think we can move this repeating code for responding to requests into `SendMessage`, so that it will be easier for people who will add new ProcessXXX functions.

Reply:
Created a overloaded version of `SendMessage` function which takes extra parameter `from` peer info.